ekncontent/Makefile.am

```diff
@@ -0,0 +1,97 @@
+## Process this file with automake to produce Makefile.in
+
+# Copyright 2016 Endless Mobile, Inc.
+
+## -----------
+## Makefile.am
+## -----------
+## Please keep this file well-commented.
+
+# Our Autoconf macros are kept in the m4/ directory
+ACLOCAL_AMFLAGS = -I m4
+
+# Forward declare variables we'll be modifying all over
+CLEANFILES =
+DISTCLEANFILES =
+EXTRA_DIST = README.md
+pkglib_LTLIBRARIES =
+
+# Compiler flags
+AM_CFLAGS = $(STRICT_CFLAGS)
+
+# # # EKN VFS LIBRARY # # #
+
+giomodule_LTLIBRARIES = libeknvfs.la
+giomoduledir = $(libdir)/gio/modules
+
+libeknvfs_la_SOURCES = \
+	lib/eknvfs/ekn-file-input-stream-wrapper.c \
+	lib/eknvfs/ekn-file-input-stream-wrapper.h \
+	lib/eknvfs/ekn-file.h \
+	lib/eknvfs/ekn-file.c \
+	lib/eknvfs/ekn-vfs.c \
+	$(NULL)
+libeknvfs_la_CPPFLAGS = \
+	@EKN_VFS_CFLAGS@ \
+	-DG_LOG_DOMAIN=\"EknVfs\" \
+	$(NULL)
+libeknvfs_la_CFLAGS = $(AM_CFLAGS)
+libeknvfs_la_LIBADD  = @EKN_VFS_LIBS@
+libeknvfs_la_LDFLAGS = \
+	-export-symbols-regex "^g_io_module_(load|unload|query)" \
+	-avoid-version \
+	$(NULL)
+
+uninstall-hook:
+	if test -z "$(DESTDIR)"; then \
```

Comment:
I'll update the hooks to properly include DESTDIR when calling gio-querymodules below. In that case it should be harmless. The cache file it generates just won't be included in any of the debian packages, and we'll used the postinst to update the real cache file.